### ISSUE TITLE
Where to place DOI?

### ISSUE THREAD

**Issue by sstrecker** [sstrecker/emisa-latex-package#26]
We have just started to assign a DOI to each published article, e. g.

DOI: 10.18417/emisa.10.1.1

Where could this information be placed in an article? 

The page header seems like a natural choice but it is already slightly crammed with information.

**Comment by sstrecker** [comment]
![mockup](https://cloud.githubusercontent.com/assets/5647896/13219993/b3bdce36-d973-11e5-9b84-3e34fd816e0e.jpg)

**Comment by sieversMartin** [comment]
Well, at the moment the DOI is placed right before the special issue title.
In the source document this looks like:

``` latex
\specialissuetitle{Domain specific language modelling}
\doi{10.18417/emisa.10.1.1}
```

**Comment by sstrecker** [comment]
Could we move the \specialissuetitle{...} below the article title in the header?

![mockup2](https://cloud.githubusercontent.com/assets/5647896/13225676/207621da-d98e-11e5-84fb-41220a06ddae.jpg)

**Comment by sieversMartin** [comment]
Changed in c2cd00d3011c998ced454b9d048af1daa1a69046

**Comment by sstrecker** [comment]
Left aligned \specialissuetitle{} on even pages: The special issue title is left aligned on even pages and should probably be right-aligned (see example).

![leftaligned](https://cloud.githubusercontent.com/assets/5647896/13250324/b4ca4d48-da29-11e5-9329-f0842bdbe0a3.jpg)

**Comment by sieversMartin** [comment]
Fixed in f8e04665de07d56c15b179a7c217713de167d94b
